sim/src/main/cc/midasexamples/Driver.cc

```diff
@@ -97,6 +97,8 @@
 #include "TerminationModule.h"
 #elif defined DESIGNNAME_CustomConstraints
 #include "CustomConstraints.h"
+#elif defined DESIGNNAME_UARTModule
```

Comment:
This isn't required, but vestigial from trying to put the test in midasexamples first?